content/blog/2016/2016-09-27-jenkins-world-2016-wrap-up-scaling.adoc

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "Jenkins World 2016 Wrap-up - Scaling"
+tags:
+- event
+- jenkinsworld
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at CloudBees.
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World 2016, role=right]
+
+I've used Jenkins for several years and been a contributor for the last two, but
+this was my first time at a Jenkins user conference.
+
+NOTE: More lead-in goes here, I'll get back to this.
+
+As I noted before, link:http://cloudbees.com[CloudBees] recorded all the sessions and will
```

Comment:
Generally I like to link to the previous blog post when referencing it in this fashion. In essence "before" would be a link to the previous post.